doc/src/manual/control-flow.md

```diff
@@ -317,8 +317,8 @@ of `&&` and `||` operators.
 
 This behavior is frequently used in Julia to form an alternative to very short `if` statements.
 Instead of `if <cond> <statement> end`, one can write `<cond> && <statement>` (which could be
-read as: <cond> *and then* <statement>). Similarly, instead of `if ! <cond> <statement> end`,
-one can write `<cond> || <statement>` (which could be read as: <cond> *or else* <statement>).
+read as: `<cond> *and then* <statement>`). Similarly, instead of `if ! <cond> <statement> end`,
+one can write `<cond> || <statement>` (which could be read as: `<cond> *or else* <statement>`).
```

Comment:
Should be like this I think?

```suggestion
read as: `<cond>` *and then* `<statement>`). Similarly, instead of `if ! <cond> <statement> end`,
one can write `<cond> || <statement>` (which could be read as: `<cond>` *or else* `<statement>`).
```

Probably worth PRing this separately.